konig-schemagen/src/main/java/io/konig/schemagen/sql/SqlTableGenerator.java

```diff
@@ -68,6 +69,23 @@ public SqlTable generateTable(Shape shape) throws SchemaGeneratorException {
 	}
 
 
+	private void addIdColumn(Shape shape, SqlTable table) {
+		SqlKeyType keyType = SqlKeyType.PRIMARY_KEY;
+		if(shape.getNodeKind() == NodeKind.IRI){
+			for (PropertyConstraint p : shape.getProperty()) {
+				if (p.getStereotype() != null) {
+					if(p.getStereotype().getLocalName().contains("primaryKey")){
```

Comment:
It is a bad practice to operate on local names.  You should add a static field named `primaryKey` of type `URI` to `io.konig.core.vocab.Konig` and compare the full URI -- not just the local name.